Sources/App/Views/PackageController/Builds/BuildShow+View.swift

```diff
@@ -75,20 +75,25 @@ private extension Build.Status {
         switch self {
             case .ok: return "Build Succeeded"
             case .failed: return "Build Failed"
+            case .pending: return "Build Pending"
         }
     }
 
     var cssClass: String {
         switch self {
             case .ok: return "green"
             case .failed: return "red"
+            // FIXME: what class to use?
+            case .pending: return "grey"
         }
     }
 
     var cssIcon: String {
         switch self {
             case .ok: return "matrix_succeeded"
             case .failed: return "matrix_failed"
+            // FIXME: what icon to use?
+            case .pending: return "matrix_pending"
```

Comment:
Dto

Reply:
Yep, this one certainly needs handling. I've set it to the clock icon we already have in 534ef6e. I might change it up later but that icon will do to get this PR merged.